{
  "affected": [
    {
      "database_specific": {
        "source": "https://storage.googleapis.com/cve-osv-conversion/osv-output/CVE-2018-19108.json"
      },
      "ranges": [
        {
          "events": [
            {
              "introduced": "0"
            },
            {
              "last_affected": "fa449a4d2c58d63f0d75ff259f25683a98a44630"
            }
          ],
          "repo": "https://github.com/exiv2/exiv2",
          "type": "GIT"
        }
      ],
      "versions": [
        "v0.10",
        "v0.11",
        "v0.12",
        "v0.13",
        "v0.14",
        "v0.15",
        "v0.16",
        "v0.16-pre1",
        "v0.17",
        "v0.17.1",
        "v0.18",
        "v0.18-pre1",
        "v0.18-pre2",
        "v0.18.1",
        "v0.18.2",
        "v0.19",
        "v0.20",
        "v0.21",
        "v0.21.1",
        "v0.22",
        "v0.23",
        "v0.23.1",
        "v0.24",
        "v0.25",
        "v0.26",
        "v0.3",
        "v0.4",
        "v0.5",
        "v0.6",
        "v0.6.1",
        "v0.6.2",
        "v0.7",
        "v0.8",
        "v0.9",
        "v0.9.1"
      ]
    }
  ],
  "details": "In Exiv2 0.26, Exiv2::PsdImage::readMetadata in psdimage.cpp in the PSD image reader may suffer from a denial of service (infinite loop) caused by an integer overflow via a crafted PSD image file.",
  "id": "CVE-2018-19108",
  "modified": "2025-10-21T04:32:33.031693Z",
  "published": "2018-11-08T08:29:00Z",
  "references": [
    {
      "type": "ADVISORY",
      "url": "https://access.redhat.com/errata/RHSA-2019:2101"
    },
    {
      "type": "FIX",
      "url": "https://github.com/Exiv2/exiv2/issues/426"
    },
    {
      "type": "FIX",
      "url": "https://github.com/Exiv2/exiv2/pull/518"
    },
    {
      "type": "ARTICLE",
      "url": "https://lists.debian.org/debian-lts-announce/2019/02/msg00038.html"
    },
    {
      "type": "ARTICLE",
      "url": "https://lists.debian.org/debian-lts-announce/2023/01/msg00004.html"
    },
    {
      "type": "ADVISORY",
      "url": "https://usn.ubuntu.com/4056-1/"
    },
    {
      "type": "WEB",
      "url": "http://lists.opensuse.org/opensuse-security-announce/2020-04/msg00009.html"
    }
  ],
  "related": [
    "ALSA-2020:1577",
    "RLSA-2020:1577",
    "SUSE-SU-2020:0921-1",
    "openSUSE-SU-2020:0482-1",
    "openSUSE-SU-2024:12399-1"
  ],
  "schema_version": "1.7.3",
  "severity": [
    {
      "score": "CVSS:3.1/AV:N/AC:L/PR:N/UI:R/S:U/C:N/I:N/A:H",
      "type": "CVSS_V3"
    }
  ]
}